ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -154,6 +154,20 @@ tests:
     - chain: openshift-e2e-test-network-qe
     workflow: openshift-e2e-aws-ovn
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-metal-ovn-fdp-qe-ipv6
+  optional: true
+  cluster: build05
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      EXTRA_MG_ARGS: --host-network
```

Comment:
@anuragthehatter What is the purpose of this flag?

Reply:
that arg is actually to allow the must-gather step at the end of the job to use host-network (whatever that means) to collect it's logs. it's supposed to be helpful in the cases when networking is having major issues and allows us to get more meaningful logs to debug the problems.